### PR TITLE
Add city names

### PR DESCRIPTION
**Pull request by stillsheryl**
This PR will:
- Update names in the dropdown menu to be the city name with airport code instead of airport name.
- Make "Book Your Flight," "Airbnb," and "Hotel" links open in a new page.
- Alphabetize city names

- [x] All Tests are passing

### CODE REVIEW

**Review by GeorgieGirl24 — APPROVED** [review]

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -63,11 +63,11 @@ function myMove_left() {
...
-  <%= f.select :departure_airport, [["Denver International", "DEN"],
-                                    ["Hartsfield/Jackson International", "ATL"],
-                                    ["Los Angeles International", "LAX"],
-                                    ["Chicago O'Hare International", "ORD"],
-                                    ["Kennedy International", "JFK"]] %>
+  <%= f.select :departure_airport, [["Denver (DIA)", "DEN"],
+                                    ["Atlanta (ATL)", "ATL"],
+                                    ["Los Angeles (LAX)", "LAX"],
+                                    ["Chicago (ORD)", "ORD"],
+                                    ["New York (JKF)", "JFK"]] %>
```
Comment (dunlapww) [inline app/views/welcome/index.html.erb:70]: New York (JFK), rather than New York (JKF)

**Review by SageOfCode — APPROVED** [review]
Awesome!

**Review by dunlapww — CHANGES_REQUESTED** [review]
Thanks for doing this Sheryl! Maybe just correct the JFK spelling and all looks good!

File: app/views/welcome/index.html.erb (hunk 1 above)
Comment (stillsheryl) [reply app/views/welcome/index.html.erb:70]: Great catch!

**Review by dunlapww — APPROVED** [review]
Great job!

**Comment by nmking22** [comment]
Great work!

**Pull request merged**